tiledb/bioimg/converters/base.py

```diff
@@ -180,16 +186,70 @@ def from_tiledb(
 
-        slide = TileDBOpenSlide(input_path, attr=attr)
-        writer = cls._ImageWriterType(output_path)
-        with slide, writer:
-            writer.write_group_metadata(slide.properties)
-            for level in slide.levels:
-                if level < level_min:
-                    continue
-                level_image = slide.read_level(level, to_original_axes=True)
-                level_metadata = slide.level_properties(level)
-                writer.write_level_image(level, level_image, level_metadata)
+        with tiledb.scope_ctx(config):
+            vfs_use = output_path.startswith("s3://")
+            if vfs_use:
+                vfs = tiledb.VFS()
+                destination_uri = vfs.open(output_path, "wb")
+            else:
+                destination_uri = output_path
+
+            writer = cls._ImageWriterType(destination_uri)
+            input_group = tiledb.Group(input_path)
+
+            with input_group, writer:
+                raw_group_meta = dict(input_group.meta)
+                assert raw_group_meta == input_group.meta
+                metadata = jsonpickle.loads(input_group.meta["metadata"])
+                writer.write_group_metadata(raw_group_meta)
+                for idx, (member, axes_metadata) in enumerate(
+                    zip(input_group, metadata.get("axes", []))
+                ):
+                    if idx < level_min:
+                        continue
+
+                    with tiledb.open(member.uri, mode="r") as array:
+                        attr_filter = array.attr(attr).filters[0]
+                        raw_array_meta = dict(array.meta)
+
+                        if isinstance(attr_filter, tiledb.WebpFilter):
+                            if (
+                                attr_filter.input_format
+                                == tiledb.cc.WebpInputFormat.WEBP_RGB
+                                or attr_filter.input_format
+                                == tiledb.cc.WebpInputFormat.WEBP_BGR
+                            ):
+                                axes_mapper = (
+                                    Axes("".join(axes_metadata.get("originalAxes")))
+                                    .webp_mapper(3)
+                                    .inverse
+                                )
+                            else:
+                                axes_mapper = (
+                                    Axes("".join(axes_metadata.get("originalAxes")))
+                                    .webp_mapper(4)
+                                    .inverse
+                                )
+                        else:
+                            axes_mapper = Axes(
+                                "".join(axes_metadata.get("storedAxes"))
+                            ).mapper(Axes("".join(axes_metadata.get("originalAxes"))))
+
+                    dask_array = dask.array.from_tiledb(member.uri, attribute=attr)
```

Comment:
Why is dask being used here?

Reply:
It was a temporary fix to export large images locally and to be removed.

Reply:
We cannot introduce dask here for that purpose. We need another solution. Dask dependency is not something we should add just for this. If you want chunked writes we can handle that.

cc: @ihnorton

Reply:
We have already a ticket for that, we should plan it and implement it similar to `chunked` ingestion. For now the way we handle the intermediate export with the configurable memory on pods and the bigger memory in local machines will not create problems for most of the files when memory is configured correctly.